### PR TITLE
Fix playlist tests (fix #23872)

### LINKED ISSUE #23872: TestDownload.test_YoutubePlaylist broken
## Checklist

- [x] I'm reporting a broken site support
- [x] I've verified that I'm running youtube-dl version **2020.01.24**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar issues including closed ones

## Verbose log

N/A 

## Description

`YoutubePlaylist` test is broken. `YoutubePlaylists` works.

```
$ python3 test/test_download.py TestDownload.test_YoutubePlaylist
[youtube:playlist] PLwiyx1dc3P2JR9N8gQaQN_BCvlSlap7re: Downloading webpage
ERROR: This playlist does not exist.
Traceback (most recent call last):
  File "/home/jx/Documents/youtube-dl/youtube_dl/YoutubeDL.py", line 796, in extract_info
    ie_result = ie.extract(url)
  File "/home/jx/Documents/youtube-dl/youtube_dl/extractor/common.py", line 530, in extract
    ie_result = self._real_extract(url)
  File "/home/jx/Documents/youtube-dl/youtube_dl/extractor/youtube.py", line 2815, in _real_extract
    has_videos, playlist = self._extract_playlist(playlist_id)
  File "/home/jx/Documents/youtube-dl/youtube_dl/extractor/youtube.py", line 2738, in _extract_playlist
    raise ExtractorError(message, expected=True)
youtube_dl.utils.ExtractorError: This playlist does not exist.

E
======================================================================
ERROR: test_YoutubePlaylist (__main__.TestDownload):
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jx/Documents/youtube-dl/youtube_dl/YoutubeDL.py", line 796, in extract_info
    ie_result = ie.extract(url)
  File "/home/jx/Documents/youtube-dl/youtube_dl/extractor/common.py", line 530, in extract
    ie_result = self._real_extract(url)
  File "/home/jx/Documents/youtube-dl/youtube_dl/extractor/youtube.py", line 2815, in _real_extract
    has_videos, playlist = self._extract_playlist(playlist_id)
  File "/home/jx/Documents/youtube-dl/youtube_dl/extractor/youtube.py", line 2738, in _extract_playlist
    raise ExtractorError(message, expected=True)
youtube_dl.utils.ExtractorError: This playlist does not exist.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "test/test_download.py", line 159, in test_template
    force_generic_extractor=params.get('force_generic_extractor', False))
  File "/home/jx/Documents/youtube-dl/youtube_dl/YoutubeDL.py", line 819, in extract_info
    self.report_error(compat_str(e), e.format_traceback())
  File "/home/jx/Documents/youtube-dl/youtube_dl/YoutubeDL.py", line 624, in report_error
    self.trouble(error_message, tb)
  File "/home/jx/Documents/youtube-dl/youtube_dl/YoutubeDL.py", line 594, in trouble
    raise DownloadError(message, exc_info)
youtube_dl.utils.DownloadError: ERROR: This playlist does not exist.

----------------------------------------------------------------------
Ran 1 test in 0.549s

FAILED (errors=1)
```

### PR DESCRIPTION
**Pull request by jxu**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/ytdl-org/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/ytdl-org/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/ytdl-org/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Bug fix
- [ ] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

fix #23872 by
- remove non-existing 'ytdl test PL' https://www.youtube.com/playlist?list=PLwiyx1dc3P2JR9N8gQaQN_BCvlSlap7re
- change 'YDL_Empty_List' https://www.youtube.com/playlist?list=PLtPgu7CB4gbZDA7i_euNxn75ISqxwZPYx to existing playlist
- fix incorrect playlist_count for https://www.youtube.com/playlist?list=PLwP_SiAcdui0KVebT0mU9Apz359a4ubsC

### CODE REVIEW

File: youtube_dl/extractor/youtube.py [hunk 1]
```diff
@@ -2495,20 +2495,14 @@ class YoutubePlaylistIE(YoutubePlaylistBaseInfoExtractor):
     _VIDEO_RE = _VIDEO_RE_TPL % r'(?P<id>[0-9A-Za-z_-]{11})'
     IE_NAME = 'youtube:playlist'
     _TESTS = [{
-        'url': 'https://www.youtube.com/playlist?list=PLwiyx1dc3P2JR9N8gQaQN_BCvlSlap7re',
+        'url': 'https://www.youtube.com/playlist?list=PLAEvfQ7P06c-DVegRvdqVcIaW2nR5ZJMO',
```
Comment (dstftw) [inline youtube_dl/extractor/youtube.py]: Replace with https://www.youtube.com/playlist?list=PL4lCao7KL_QFodcLWhDpGCYnngnHtQ-Xf.

File: youtube_dl/extractor/youtube.py [hunk 2]
```diff
@@ -2495,20 +2495,14 @@ class YoutubePlaylistIE(YoutubePlaylistBaseInfoExtractor):
     _VIDEO_RE = _VIDEO_RE_TPL % r'(?P<id>[0-9A-Za-z_-]{11})'
     IE_NAME = 'youtube:playlist'
     _TESTS = [{
-        'url': 'https://www.youtube.com/playlist?list=PLwiyx1dc3P2JR9N8gQaQN_BCvlSlap7re',
```
Comment (dstftw) [inline youtube_dl/extractor/youtube.py]: Replace with https://www.youtube.com/playlist?list=PL4lCao7KL_QFVb7Iudeipvc2BCavECqzc.

Comment (jxu) [reply youtube_dl/extractor/youtube.py]: This is a non-empty playlist - which test is it for?

Comment (jxu) [reply youtube_dl/extractor/youtube.py]: I will add back in original test.

**Pull request merged**